README.md

```diff
@@ -43,6 +45,9 @@ Create a For Loop that outputs the following string for each piece of fruit in t
 
 ```js
 // write code here
+for (var i = 0; i < foods[0].length; i++) {
+  console.log("I want to eat a " + foods[0][i]);
+}
```

Comment:
It says for each piece of fruit. foods[1] doesn't consist of fruits!

On Mon, Jun 13, 2016 at 1:47 PM, Adrian Maseda <notifications@github.com>
wrote:

> In README.md
> <https://github.com/ga-wdi-quizzes/week-03/pull/53#discussion_r66835830>:
>
> > @@ -43,6 +45,9 @@ Create a For Loop that outputs the following string for each piece of fruit in t
> >
> >  ```js
> >  // write code here
> > +for (var i = 0; i < foods[0].length; i++) {
> > +  console.log("I want to eat a " + foods[0][i]);
> > +}
>
> This would only iterate through (part of) the first array inside of foods.
> To print out each fruit individually, you would need to nest another for
> loop inside of your existing one.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ga-wdi-quizzes/week-03/pull/53/files/5e02a15e549da616b024f016678cf4b1315b6ece#r66835830>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACIF2K-2QTMJJKTMngwuENyFqCe7a5yTks5qLZe1gaJpZM4Iz6VN>
> .
>